Security/Authentication/User/User.php

```diff
@@ -39,8 +39,4 @@ public function getUserIdentifier(): string
     {
         return $this->userIdentifier ?? 'identifier';
     }
-
-    public function eraseCredentials(): void
-    {
-    }
```

Comment:
I'm OK with something like this.

```suggestion

    /**
     * @psalm-suppress PossiblyUnusedMethod
     *
     * Kept for backwards compatibility with Symfony <8
     */
    public function eraseCredentials(): void
    {
    }
```

Reply:
@wazelin adjusted - thanks!